components/decimal/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_decimal"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "benches/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[package.metadata.cargo-all-features]
+skip_optional_dependencies = true
+
+[dependencies]
+smallstr = { version = "0.2", features = ["serde"] }
```

Comment:
Did you evaluate it against `ArrayString`? `ArrayString` is part of `ArrayVec` crate which is well maintained, aligned with upcoming stack array stdlib extension and was faster than `smallstr` in my benchmarks - https://github.com/zbraniecki/tinystr/wiki/Performance#performance-vs-other-small-string-libraries

Reply:
Yes, I did this evaluation and we discussed it last June. I don't know if things have changed since then. #113 is the issue, and it's also documented in the style guide.

Reply:
Ah, I didn't realize you need heap spilling. Yeah, `smallstr` may be a good choice for now. I do hope that we'll get a next-gen stack+heap string based on the stack array that is planned for stdlib.